<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addnewmovie</class>
 <widget class="QWidget" name="addnewmovie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">addnewmovie
{
	background-color: rgb(0, 0, 0);
}
QLineEdit
{
	background-color: rgb(237, 212, 0);
	color: rgb(0, 0, 0);
	border-radius:12px;
	font: 63 italic 16pt &quot;Go&quot;;
}
QLineEdit:hover
{
	background-color: rgb(252, 233, 79)
}
QPushButton#btn_cancel
{
	color: rgb(164, 0, 0);
	background-color: rgb(0, 0, 0);
	border-radius:12px;
	border:2px solid red;
}
QPushButton:hover#btn_cancel
{
	color: rgb(238, 238, 236);
	background-color: rgb(164, 0, 0);
}
QPushButton#btn_apply
{
	color: rgb(78, 154, 6);
	background-color: rgb(0, 0, 0);
	border-radius:12px;
	border:2px solid green;
}
QPushButton:hover#btn_apply
{
	color: rgb(238, 238, 236);
	background-color: rgb(78, 154, 6);
}</string>
  </property>
  <widget class="QPushButton" name="btn_apply">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>590</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="1" column="0">
     <widget class="QLineEdit" name="get_moviename">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="get_writer">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_producer">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);</string>
      </property>
      <property name="text">
       <string>Producer</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_cast">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>6</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);</string>
      </property>
      <property name="text">
       <string>Cast</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_contries">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);</string>
      </property>
      <property name="text">
       <string>Countries</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_genres">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);</string>
      </property>
      <property name="text">
       <string>Genres</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_moviename">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);</string>
      </property>
      <property name="text">
       <string>Movie's Name</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="get_cast2">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_director">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);</string>
      </property>
      <property name="text">
       <string>Director</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="get_director">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="get_cast3">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_tickets">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);</string>
      </property>
      <property name="text">
       <string>Tickets</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="get_producer">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_writer">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);</string>
      </property>
      <property name="text">
       <string>Writer</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_releaseyear">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);</string>
      </property>
      <property name="text">
       <string>Release Year</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="get_cast1">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="get_musician">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="get_countries">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="get_releaseyear">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="get_genres">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="get_tickets">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLineEdit" name="get_sans">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Go</family>
        <pointsize>16</pointsize>
        <weight>7</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_sans">
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);</string>
      </property>
      <property name="text">
       <string>Sans</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_musician">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);</string>
      </property>
      <property name="text">
       <string>Musician</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>590</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
